code/dataobjects/Comment.php

```diff
@@ -63,14 +67,13 @@ class Comment extends DataObject {
 		'Created',
 		'BaseClass',
 	);
-	
+
 	private static $summary_fields = array(
 		'Name' => 'Submitted By',
 		'Email' => 'Email',
 		'Comment' => 'Comment',
 		'Created' => 'Date Posted',
-		'ParentTitle' => 'Parent',
-		'IsSpam' => 'Is Spam',
```

Comment:
If we are going to show these posts in the "New Comments" list, then how are we going to know which new comments were automatically detected as spam?